CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Version changelog
 
+## 0.18.0
+
+Bugfixes:
+
+* Fix Databricks OAuth M2M on Azure ([#513](https://github.com/databricks/databricks-sdk-py/pull/513)).
+
+Other noteworthy changes:
+
+* Use `[]` instead of `None` as default list value for deserialising responses ([#361](https://github.com/databricks/databricks-sdk-py/pull/361)).
+* Support dev and staging workspaces ([#514](https://github.com/databricks/databricks-sdk-py/pull/514)).
+
+API Changes:
+
+ * Added `exists()` method for [w.tables](https://databricks-sdk-py.readthedocs.io/en/latest/workspace/tables.html) workspace-level service.
+ * Added [w.lakehouse_monitors](https://databricks-sdk-py.readthedocs.io/en/latest/workspace/lakehouse_monitors.html) workspace-level service.
+ * Added `databricks.sdk.service.catalog.CreateMonitor` dataclass.
```

Comment:
We can club them together to make it changelog friendly for example:
Added the following dataclasses: `databricks.sdk.service.catalog.CreateMonitor`, `databricks.sdk.service.catalog.DeleteLakehouseMonitorRequest` ...

Ref: https://github.com/databricks/databricks-sdk-py/pull/502. Stamping to unblock release.